### PR TITLE
feat: Convert all issue templates into issue forms

### PR DESCRIPTION
**Pull request by Panquesito7**
Things added/changed:

- Convert all issue templates into issue forms.
- Disable blank issues, but provide an `Other` generic template for all kinds of issues.
  - The `Other` template is taken from some of @nhcarrigan's projects. Thanks, Nick! 😄

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug.yml [hunk 1]
```diff
@@ -0,0 +1,35 @@
+name: Bug report
+description: Create a report to help us improve. Report bugs found while using the project
+title: "[BUG] <description>"
+labels: ["💣 Bug"]
```
Comment (nhcarrigan) [inline .github/ISSUE_TEMPLATE/bug.yml:4]:
```suggestion
labels: ["🛠 goal: fix"]
```

File: .github/ISSUE_TEMPLATE/feature.yml [hunk 2]
```diff
@@ -0,0 +1,35 @@
+name: Feature request
+description: Suggest features, propose improvements, discuss new ideas
+title: "[FEATURE] <description>"
+labels: ["⭐ goal addition"]
```
Comment (nhcarrigan) [inline .github/ISSUE_TEMPLATE/feature.yml:4]:
```suggestion
labels: ["⭐ goal: addition"]
```

File: .github/ISSUE_TEMPLATE/other.yml [hunk 3]
```diff
@@ -0,0 +1,22 @@
+name: Other
+description: Use this for any other issues. PLEASE do not create blank issues
+title: "[OTHER]"
+labels: [triage]
```
Comment (nhcarrigan) [inline .github/ISSUE_TEMPLATE/other.yml:4]:
```suggestion
labels: ["🚦 status: awaiting triage"]
```

**Review by nhcarrigan — CHANGES_REQUESTED** [review]
Need the correct labels here.

**Review by nhcarrigan — APPROVED** [review]
Looks good to me

**Review by eddiejaoude — APPROVED** [review]
Great team work 💪

**Pull request merged**